docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,98 @@
+
+MUI Toolpad aims to reduce the overhead associated with frontend development by providing visual editors and reducing the amount of boilerplate required to bring data on the page. With basic frontend knowledge you can build an app in just one sitting. On this page, we'll try to address your concerns by covering all aspects of building an admin panel. This shall help you evaluate if Toolpad is the right product for you.
+
+## TLDR
+
+- **Integrates well with your existing code**. You can reuse your database models, client libraries, secrets, bespoke components
+- Integrates well with your **development lifecycle.** Your project lives on your own file system. You can use your own IDE, version control system, deployment target, CLI tools
+- Comes with a **Drag and drop builder** to simplify UI building
+- Sets **Healthy UI constraints** that prevent common pitfalls and anti-patterns
+- Lets you maintain a **single source of truth** for all business logic
+- **Less boilerplate** so you can focus on the important parts of the app
+
+## Admin builders market overview
+
+The main problem with the tools currently in the Node.js and TypeScript ecosystem is that they require a major tradeoff between flexibility and speed. For a back office application developer, the backend and the database schema is an area of comfort. What's relatively challenging is building the UI and having a reasonable UX that can show the required data in a readable and editable format. And that's where all the admin builders come into the picture. If we closely look at the market, we can divide them into the following categories:
+
+### Fast development, less flexibility
+
+Most modern internal tool builders in the market fall into this category. Products like Retool, Appsmith act like a SaaS and provide a UI for rapid application development. The apps can be either hosted by you or them. They are excellent products for someone who's looking to build fast. The caveats are that you can't use your IDE, your existing codebase, or logic, and it can cost a lot as you scale.
+
+### Moderate development speed, moderate flexibility
+
+Products like airplane.dev, interval.com fall under this category. They have APIs for all admin use cases; while this can be great for those who enjoy writing code, it is a bit too much as adding a component also requires writing code. Their main advantage over previous category is that they store their configuration in local code files. This puts more control over the development lifecycle back in the developer's hands. Users can version control, deploy, and use any IDE they want.
+
+### Slow development speed, more flexibility
+
+Products like react admin, refine.dev fall in this category, they are all code and have APIs for everything to build an admin panel from scratch. They don't support cloud hosting and are a relatively cheaper alternative.
+
+We'll do a detailed comparison with these tools in the future.
```

Comment:
Why mention this? We can either do the comparison now and add it, or just add the comparison later?